pom.xml

```diff
@@ -524,6 +524,23 @@
            </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>2.1.0</version>
+        <configuration>
+          <to>
+            <image>docker.io/plos/rhino:${project.version}</image>
```

Comment:
we aren't always great about keeping this up-to-date. I assume this will mean that we need to be careful to keep these always up-to-date?

Reply:
You would just need to update the project version when you wanted the docker image tagged differently. We can also add any number of tag steps after the build if we want (outside of maven).

Reply:
Sounds like we'll need to maintain good hygiene and ensure that the `develop` branch always uses `-SNAPSHOT` suffixes and the non-snapshot branches are always on the release brancehs only. That's fine, we should be able to do that.